conf/messages

```diff
@@ -348,8 +348,8 @@ error.line2_invalid_characters=Your address can include letters a to z, numbers,
 error.line3_invalid_characters=Your address can include letters a to z, numbers, spaces, hyphens (-), apostrophes (‘), commas (,), forward slashes (/) , full stops (.) and ampersands (&).
 error.line4_invalid_characters=Your address can include letters a to z, numbers, spaces, hyphens (-), apostrophes (‘), commas (,), forward slashes (/) , full stops (.) and ampersands (&).
 error.line5_invalid_characters=Your address can include letters a to z, numbers, spaces, hyphens (-), apostrophes (‘), commas (,), forward slashes (/) , full stops (.) and ampersands (&).
-head.error.enter_a_valid_uk_postcode=Enter a valid UK postcode
-error.enter_a_valid_uk_postcode=Enter a valid UK postcode
+head.error.enter_a_valid_uk_postcode=Enter a postcode
+error.enter_a_valid_uk_postcode=Enter a postcode
```

Comment:
Can we just delete one of these keys if they are the same

Reply:
All the views are using the same component for the error summary on top of the page which adds the head. prefix to the error message, if I change it to not use that it will affect other messages and screens that were not being changed in this audit